examples/login-oidc/client_config_idtoken.js

```diff
@@ -0,0 +1,38 @@
+  // Don't commit this file to your public repos. This config is for first-run
+  //
+
+  exports.creds = {
+    // This is your app's 'REPLY URL' in AAD
+    returnURL: 'http://localhost:3000/auth/openid/return',
+
+    // replace <your_tenant_name> with your tenant name,
+    // which is something like: "*.onmicrosoft.com"
+    identityMetadata: 'https://login.microsoftonline.com/<your_tenant_name>/.well-known/openid-configuration',
```

Comment:
it might be worth mentioning here in the comment that this can also be the tenant guid id as well. some examples use the guid and so it can be confusing which is valid when both are.